src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonObject.IDictionary.cs

```diff
@@ -84,25 +74,12 @@ public void Clear()
         /// <exception cref="ArgumentNullException">
         ///   <paramref name="propertyName"/> is <see langword="null"/>.
         /// </exception>
-        public bool ContainsKey(string propertyName)
-        {
-            InitializeIfRequired();
-            Debug.Assert(_dictionary != null);
-            return _dictionary.ContainsKey(propertyName);
-        }
+        public bool ContainsKey(string propertyName) => Dictionary.ContainsKey(propertyName);
 
         /// <summary>
         ///   Gets the number of elements contained in <see cref="JsonObject"/>.
         /// </summary>
-        public int Count
-        {
-            get
-            {
-                InitializeIfRequired();
-                Debug.Assert(_dictionary != null);
-                return _dictionary.Count;
-            }
-        }
+        public int Count => Dictionary.Count;
```

Comment:
We wouldn't want to just return 0 here if it was uninitialized rather than forcing initialization?

Reply:
Nevermind, I misunderstood what initialization was doing.